src/pages/SelectedVideoPage/VideoPage.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useRef, useEffect } from "react";
+import { useSelector } from "react-redux";
+import { useParams } from "react-router-dom";
+import ReactPlayer from "react-player/youtube";
+import styles from "./VideoPage.module.css";
+import { VideoSuggestionCard } from "components";
+import { videoShufflerFunction } from "utilities";
+
+const VideoPage = () => {
+  const { videoId } = useParams();
+  const { videosList } = useSelector((storeReceived) => storeReceived.videos);
+  const { sidebarView } = useSelector((storeReceived) => storeReceived.sidebar);
+  const videoSelected = videosList.find(
+    (everyVideo) => everyVideo._id === videoId
+  );
+  const { title, dateOfUpload, viewQuantity, description } = videoSelected;
+  const videoPageRef = useRef();
+  const [videoPageDimensions, setVideoPageDimensions] = useState();
+  const getCurrentPageDimensions = () => {
+    const widthReceived = videoPageRef.current.clientWidth;
+    setVideoPageDimensions(widthReceived);
+  };
+  //   For receiving the page width whenever the window is resized and setting it
+  useEffect(() => {
+    getCurrentPageDimensions();
+  }, [videosList, sidebarView]);
+  useEffect(() => {
+    window.addEventListener("resize", getCurrentPageDimensions);
+  }, []);
+  return (
+    <div
+      ref={videoPageRef}
+      className={`${styles.video_page_container}`}
+      style={{ flexDirection: videoPageDimensions < 1165 ? "column" : "row" }}
+    >
+      {/* Main selected video content would be displayed here */}
+      <div
+        className={`${styles.selected_video_section} g-flex-column`}
+        style={{
+          alignSelf: videoPageDimensions < 1165 && "stretch",
+        }}
+      >
+        <div className={`${styles.selected_video_player}`}>
+          <ReactPlayer
+            width="100%"
+            height="100%"
+            url={`https://youtube.com/watch?v=${videoId}`}
+            controls
```

Comment:
You can also set `playing=true` to start the media when this page is opened. So that the user won't have to click on the video again to play it.